test/api.js

```diff
@@ -4,9 +4,10 @@
 
 const url = require('url');
 const assert = require('insist');
-const bidcrypto = require('browserid-crypto');
 const nock = require('nock');
 const buf = require('buf').hex;
+const generateRSAKeypair = require('generate-rsa-keypair');
```

Comment:
i don't anticipate us generating prod keys with it so i for one don't care all that much, but yeah it just a wrapper around the one openssl function.